gulpfile.babel.js

```diff
@@ -2,9 +2,9 @@ const babel = require('gulp-babel');
 const del = require('del');
 const gulp = require('gulp');
 const eslint = require('gulp-eslint');
-const replace = require("gulp-replace");
-const uglify = require("gulp-uglify");
-const packageJson = require("./package.json");
+const replace = require('gulp-replace');
+const uglify = require('gulp-uglify');
+const packageJson = require('./package.json');
```

Comment:
It is a little hard to pick out the relevant changes in this diff among all of the quote-style changes.

These are not related to the upgrade, right?

Reply:
Right. The file used a mix of double-quoted strings and single-quoted strings. I just converted all of them to single-quoted strings.

Reply:
Minimal, targeted diffs generally make life easier for reviewers who are looking at your code.

I like that you want to clean the quote situation up in here, but in the future please do this sort of thing in a dedicated patch.